packages/snap/src/types/proof-types.ts

```diff
@@ -22,6 +22,10 @@ export type CreateProofRequest = {
   query: ProofQuery;
 };
 
+export type CreateProofRequestParams = {
+  issuerDID: string;
```

Comment:
```suggestion
  issuerDid: string;
```

For consistency sake